iceoryx_hoofs/cmake/IceoryxPlatform.cmake

```diff
@@ -44,19 +44,28 @@ else()
 endif()
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-    set(ICEORYX_WARNINGS PRIVATE ${ICEORYX_WARNINGS} /W0) # TODO iox-#33 set to /W1
+    set(ICEORYX_C_WARNINGS PRIVATE ${ICEORYX_C_WARNINGS} /W0) # TODO iox-#33 set to /W1
```

Comment:
Why the adding `${ICEORYX_C_WARNINGS}` to itself. they are defined here and should be empty at this point. In the best case it's a non existing variable and cmake ignores it. In the worst case multiple cmake invokations will add the same warnings all over again to the variable and in the absolute worst case, an cmake update will result in printing warnings because we use an undefined variable. We already had this and needed to create a patch release for ROS.